rpcserverhandlers_test.go

```diff
@@ -636,6 +644,26 @@ func cloneParams(params *chaincfg.Params) *chaincfg.Params {
 	return &paramsCopy
 }
 
+// block432100 mocks block 432,100 of the block chain.  It is loaded and
+// deserialized immediately here and then can be used throughout the tests.
+var block432100 = func() wire.MsgBlock {
+	// Load and deserialize the test block.
+	blockHexBytes, err := ioutil.ReadFile("internal/rpcserver/testdata/block432100.hex")
```

Comment:
This should use `filepath.Join` so it's not OS specific.  See [here](https://github.com/decred/dcrd/blob/b60c60ffe98bcea5becc4e9e4d2a4fe6152401f6/database/ffldb/whitebox_test.go#L35) for example.

Also, since it's a separate file, it's probably worth compressing the raw bytes with bzip2 as is the case in the linked database code.

The function would barely change.  eg:

```Go
//  ... Up near the top so it can be reused if more tests want to use a fixture....
var testDataPath = filepath.Join("internal", "rpcserver", "testdata")

// ...
var block432100 = func() wire.MsgBlock {
	// Load and deserialize the test block.
	blockDataFile := filepath.Join(testDataPath, "block432100.bz2")
	fi, err := os.Open(blockDataFile)
	if err != nil {
		panic(err)
	}
	defer fi.Close()
	var block wire.MsgBlock
	err = block.Deserialize(bzip2.NewReader(fi))
	if err != nil {
		panic(err)
	}
	return block
}()
```

You can create the compressed file with:

```bash
$ dcrctl getblockhash 432100 | dcrctl getblock - false | xxd -r -p | bzip2 > block432100.bz2
```

If you prefer to leave it as hex, that's fine too.  I do generally prefer to compress data files though to reduce the overall size of the repository.